app/processing/due-dates/get-payment-schedule.js

```diff
@@ -17,6 +17,8 @@ const getPaymentSchedule = (schedule, dueDate, settledValue, totalValue, current
     case Y2:
       // Y2 schedule to be one payment on due date, second four months later.
```

Comment:
is the comment required?

Reply:
I would say yes and let me tell you why...

The other schedules we have are fairly clear. Q4 - split across four quarters. M12 - split across 12 months. Y2 you'd possibly think is just split across two halves of the year, ie, one half at due date and one half six months later.

This is not how Y2 works, rather second part of payment being four months later.

Frankly I haven't much faith that I'll remember why we've coded it the way we have in a few weeks time, let alone anyone else. So would like to be clear why its coded that way just in case anything goes wrong

Reply:
No, good enough. If there's justification for it over the others it's all good with me. Please collect your gif below.